tests/test_utils_other.py

```diff
@@ -25,18 +25,11 @@
 from datacube.utils.dates import date_sequence
 from datacube.utils.math import (
     num2numpy,
-    is_almost_int,
-    maybe_zero,
-    maybe_int,
-    snap_scale,
     valid_mask,
     invalid_mask,
-    clamp,
     unsqueeze_data_array,
     unsqueeze_dataset,
     spatial_dims,
```

Comment:
I think `odc-geo` has a `spatial_dims` method that might be useful here: https://odc-geo.readthedocs.io/en/latest/_api/odc.geo.xr.ODCExtension.spatial_dims.html#odc.geo.xr.ODCExtension.spatial_dims

Reply:
So it does! Thanks for pointing that out.